### ISSUE TITLE
problem on ipad horenzental video's taken 

### ISSUE THREAD

**Issue by geert48** [joomlaworks/allvideos#36]
Horizental video's taken with iphone of ipad are only to see on the whole leftside of the framed player
Also buttons are grey and when i activate them they are getting blue on mij ipad and iphone
Kind regards

**Comment by fevangelou** [comment]
Screenshot please.

**Comment by geert48** [comment]
Hi there
This is the screenshot
See link
Kind regards

https://imgur.com/a/vaBO9

Verstuurd vanaf mijn iPad

**Comment by fevangelou** [comment]
Does it display the same on desktop browsers? It appears the video was shot in portrait mode ("vertically") and not in landscape mode ("horizontally")...

**Comment by geert48** [comment]
Hi
It appears on portrait mode only on ipad and iphone Safari and Chrome NOT on desktop browsers

Verstuurd vanaf mijn iPad

**Comment by fevangelou** [comment]
Let me rephrase my last question. Is the video SHOT in portrait mode? In other words, was the smartphone or tablet in vertical position when shooting the video in the first place? If so, AllVideos is pre-configured to work with landscape-shot media, as those are more proper for viewing on any device.

**Comment by geert48** [comment]
Hi
They where shot in Potrait mode but how can i fix this for now?
Before this version all worked fine portrait mode and landscape mode
Kind regards

Verstuurd vanaf mijn iPad

**Comment by fevangelou** [comment]
The player has changed. That's no easy solution I'm afraid. It requires a code change on your part until I add a permanent solution to choose between using both Clappr and JW Player or JW Player only.

Edit the file /plugins/content/jw_allvideos/jw_allvideos/includes/sources.php and near line 138, next to the formats you wish to force JW Player, change and reference of `$clapprEmbed` and `$clapprEmbedRemote` to `$jwPlayerEmbed` and `$jwPlayerEmbedRemote` respectively.

This will swap Clappr for JW Player which will probably work with your portrait videos. Keep in mind this is the new version (v7) of JW Player. The previous one (v6) is now obsolete.
